site/static/diagrams/gameserver-states.dot

```diff
@@ -28,6 +28,7 @@ digraph {
     Scheduled -> RequestReady [ label="SDK.ready()", color="purple" ]
     RequestReady -> Ready [ label="ready to be allocated", color="blue" ]
     Reserved -> Ready [label="if seconds > 0  \land failed to call  \lSDK.allocate()  " color="purple"]
+    Allocated -> Ready [ label="(optional)  \lSDK.ready()", color="purple" ]
```

Comment:
Mark - does `SDK.ready()` move it directly back to the Ready state or back to the RequestReady state?

Also, I don't think the "(optional)" prefix is required, since many of the arrows in the diagram are optional.

Reply:
Yes, you are 100% right. This should point to RequestReady (which then flows down to Ready).

Agreed on (optional) prefix as well.

Otherwise, this looks like a great addition, thanks for submitting!